transmission_interface/CMakeLists.txt

```diff
@@ -19,9 +19,17 @@ find_package(TinyXML2 REQUIRED)
 add_library(transmission_parser SHARED src/transmission_parser.cpp)
 target_include_directories(transmission_parser PUBLIC include)
 ament_target_dependencies(transmission_parser hardware_interface tinyxml2_vendor TinyXML2)
-ament_export_dependencies(hardware_interface tinyxml2_vendor TinyXML2)
+ament_export_dependencies(transmission_parser tinyxml2_vendor TinyXML2)
```

Comment:
consider moving this further down next to `ament_export_include_directories`

Reply:
moved

Reply:
fixed